guides/common/modules/ref_compliance-of-rhel-hosts-in-red-hat-insights.adoc

```diff
@@ -0,0 +1,13 @@
+[id="compliance-of-rhel-hosts-in-red-hat-insights_{context}"]
+= Compliance of RHEL Hosts in Red Hat Insights
+
+For RHEL hosts with enabled Insights, you can use the Red Hat Insights compliance service to assess and monitor security policy compliance and vulnerabilities on your hosts.
```

Comment:
There'a also an attribute for "RHEL". See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L113

Reply:
I know, but I don't want to use it, because then the title and the sentence become too long. I'm using the abbreviation for the sake of readability.